sdcontent/INSTRUCTIONS-README.txt

```diff
@@ -1,11 +1,11 @@
 NOOBS INSTALLATION INSTRUCTIONS
 
-1. Insert an SD card that is 8GB or greater in size into your computer.
+1. Insert an SD card that is 16GB or greater in size into your computer.
 2. Format the SD card using the platform-specific instructions below:
    a. Windows (32GB cards and under)
       i. Download the SD Association's Formatting Tool from https://www.sdcard.org/downloads/formatter_4/eula_windows/
       ii. Install and run the Formatting Tool on your machine
-      iii. Check that the SD card you inserted matches the one selected by the Tool
+      iii. Check that the SD card you inserted matches the one selected by the tool

```

Comment:
perhaps "tool" -> "Formatting Tool" for consistency with the previous line? (and then the same change in the Mac instructions)

Reply:
Don't think so. That would make it more wordy. It should be obvious which tool this is referring to, since it was just named in the sentence which immediately precedes it.

Reply:
Actually I've changed my mind on that one.